torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -3006,16 +3331,24 @@ Error PyTorchModelLoader::loadAttributes(
           std::make_pair(&ival.toObjectRef(), newNameHierarchy);
       continue;
     } else if (ival.isTensor()) {
-      const auto &ptTensor = ival.toTensor();
-      auto glowTensor = ptTensorToGlowTensor(ptTensor);
-
-      glow::Constant *glowConstant = F_.getParent()->createConstant(
-          newNameHierarchy, std::move(glowTensor));
+      const auto ptTensor = ival.toTensor();
+      // PyTorch Tensor extracted type is kByte
+      // indicate it is the address of stored weights of quantized
+      // linear or conv.
+      if (ptTensor.scalar_type() == at::kByte) {
```

Comment:
You can check its meta data to make sure it's a specific hacked tensor by doing 
https://bddppq.github.io/codebrowser/pytorch/pytorch/aten/src/ATen/cpp_custom_type_hack.h.html#20

But it's fine for now.

Reply:
yes, the meta type that custom_hack check is a type inside fbgemm's header, which require us to call fbgemm.h again. I think this should work for now, eventually I should thinking of having a registered metatype indicating this hacked type, or look its following node to make sure it is weight of conv/linear.